server/views/pages/plan.njk

```diff
@@ -96,10 +96,12 @@
   <p class="govuk-body govuk-!-margin-top-5">
       {%- if data.plan.agreementDate !== null %}
           {%- if data.plan.agreementStatus === 'COULD_NOT_ANSWER' and data.readWrite === true %}
-              <a class='govuk-!-display-none-print' href="/update-agree-plan">{{ locale.couldNotAgreePlanLink }}</a> {{ locale.couldNotAgreePlan }}
+              <div id='update-assessment-text' class='govuk-!-display-none-print'>
```

Comment:
presumably this should be double quotes around the HTML attributes.

Reply:
Both work, the DOM reads single quotes like double quotes but I can change for consistency. Updated in [3d298f2](https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/450/commits/3d298f2d81100c370693152b3b7c9c1bad78e9b0)

Reply:
yeah it was just for consistency, I could have been clearer. thanks for the change.